reconcile/utils/acs/base.py

```diff
@@ -68,5 +69,10 @@ def generic_request(
             json=json,
             timeout=self.timeout,
         )
-        response.raise_for_status()
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            logging.error(f"{str(e)}: {response.text}")
+            raise e
```

Comment:
`raise` is enough to show the full stack trace.

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4261/commits/11ff62c1f9ef41a1be4eaa96a47b11b84d9f1230